WORKSPACE

```diff
@@ -1,11 +1,25 @@
-workspace(name = "io_bazel_rules_play_routes")
+workspace(name = "compiler_cli")
```

Comment:
I'll rename this to `play_routes_compiler_cli` before merging. Holding off on getting more feedback before pushing

Reply:
Thanks. `compiler_cli` doesn't seem quite specific enough.